core/src/main/java/com/orange/ouds/core/component/OudsDivider.kt

```diff
@@ -0,0 +1,141 @@
+/*
+ * Software Name: OUDS Android
+ * SPDX-FileCopyrightText: Copyright (c) Orange SA
+ * SPDX-License-Identifier: MIT
+ *
+ * This software is distributed under the MIT license,
+ * the text of which is available at https://opensource.org/license/MIT/
+ * or see the "LICENSE" file for more details.
+ *
+ * Software description: Android library of reusable graphical components
+ */
+
+package com.orange.ouds.core.component
+
+import androidx.compose.foundation.isSystemInDarkTheme
+import androidx.compose.foundation.layout.height
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.width
+import androidx.compose.material3.HorizontalDivider
+import androidx.compose.material3.VerticalDivider
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.tooling.preview.PreviewParameter
+import androidx.compose.ui.unit.dp
+import com.orange.ouds.core.theme.OudsTheme
+import com.orange.ouds.core.theme.value
+import com.orange.ouds.core.utilities.OudsPreview
+import com.orange.ouds.foundation.utilities.EnumPreviewParameterProvider
+import com.orange.ouds.foundation.utilities.UiModePreviews
+import com.orange.ouds.theme.tokens.OudsColorKeyToken
+
+//TODO Add DSM link when available
+// <a href="https://unified-design-system.orange.com/" class="external" target="_blank">OUDS Divider design guidelines</a>
+/**
+ * An OUDS horizontal divider.
+ *
+ * This composable renders a horizontal line that can be used to visually separate content.
+ * The color of the divider can be specified using the [OudsDivider.Color] enum, and the thickness is defined by the current theme's divider border width.
+ *
+ * @param modifier [Modifier] applied to the divider.
+ * @param color The color of the divider, chosen from the [OudsDivider.Color] enum. Defaults set to `OudsDivider.Color.Default`.
```

Comment:
I think it should be either "Defaults to \`OudsDivider.Color.Default\`" or "Default value set to \`OudsDivider.Color.Default\`".